diagnostic.md

```diff
@@ -70,15 +62,11 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 
 If we are in the `project` directory and use `cd planning`, is a relative or absolute path being referenced? How do you know?
 
- <!-- Answer Starts Here -->
-
- <!-- Answer Ends Here -->
+It is a relative path, because you can only reach planning with the command 'cd planning' if you are in the 'project' directory.
 
 3. Now suppose that we have an image file living inside our project. Would we refer to it with an absolute or relative path? Why?
 
- <!-- Answer Starts Here -->
-
- <!-- Answer Ends Here -->
+ Absolute path - because you can only refer to directories through relative paths. 
```

Comment:
Relative: the image file lives inside our current working directory, project ; as a result, an absolute path is not needed to access it.